release_notes/372-release-notes.adoc

```diff
@@ -8,4 +8,265 @@ toc::[]
+| GA
+| DEP
+| DEP
+
+| Retrieving groups by property
+| GA
+| DEP
+| DEP
+
+| `vulns` fields of `storage.Node` object in response payload of `v1/nodes` 
+| GA
+| DEP
+| DEP
+
+| `/v1/cves/suppress` and `/v1/cves/unsuppress`
+| GA
+| DEP
+| DEP
+
+| `ids` field in the `/v1/cves/suppress` and `/v1/cves/unsuppress` API payload 
+| GA
+| DEP
+| DEP
+
+| `cves.ids` field of the `storage.VulnerabilityRequest` object in the response of `VulnerabilityRequestService` endpoints
+| GA
+| DEP
+| DEP
+
+| Scanning support for Ubuntu 21.10
+| GA
+| GA
+| DEP
+
+| Permission `ClusterCVE`
+| GA
+| GA
+| DEP  
+
+| `Label` and `Annotation` search options
+| GA
+| GA
+| DEP
+
+|====
+
+[id="deprecated-features-372"]
+=== Deprecated features
+
+* Permissions for permission sets will be grouped for simplification. The following list describes the new permissions and indicates the deprecated permissions that will be removed in a future release:
+** The `Access` permission will replace the following permissions: `AuthPlugin`, `AuthProvider`, `Group`, `Licenses`, `Role`, and `User`.
+** The `DeploymentExtension` permission will replace the following permissions: `Indicator`, `NetworkBaseline`, `ProcessWhitelist`, and `Risk`.
+** The `Integration` permission will deprecate the following permissions: `APIToken`, `BackupPlugins`, `ImageIntegration`, `Notifier`, and `SignatureIntegration`.
+** The `Image` permission will replace the permission `ImageComponent`.
+* Previously, groups were retrieved by using the field `props` (`props.authProviderId`, `props.key`, `props.value`). This field will be replaced by the new `props.id` field. Use the `props.id` field to retrieve groups in the Application Programming Interface (API). Note the following:
+** Retrieval by using the `props` fields will be removed in version a future release.
+** Until removal, retrieval by using the `props` field will work if the result is unambiguous (no more than one group is found with the `props` field).
+* `/v1/cves/suppress` and `/v1/cves/unsuppress` have been deprecated and will be removed in a future release. After these are removed:
+** Use `/v1/imagecves/suppress` and `/v1/imagecves/unsuppress` to snooze and unsnooze image vulnerabilities.
+** Use `/v1/nodecves/suppress` and `/v1/nodecves/unsuppress` to snooze and unsnooze node and host vulnerabilities.
+** Use `/v1/clustercves/suppress` and `/v1/clustercves/unsuppress` to snooze and unsnooze platform (Kubernetes, Istio, and {ocp}) vulnerabilities.
+* Permission `ClusterCVE` is deprecated and will be superseded by the existing permission `Cluster`. 
+* `Label` and `Annotation` search options are deprecated and will be removed in the {product-title-short} 3.73 release. They will be replaced by the search options listed in the following table.
+
+.Search options
+[cols="1,1,1",options="header"]
+|====
+|Resource |Deprecated search option | New search option 
+
+| Node
+| Label
+| Node Label
```

Comment:
I think these can be lowercase. This it the API object labels array. For example:

```
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app.kubernetes.io/name: myservice
    app.kubernetes.io/instance: myservice-abcxzy
```

Reply:
As per the other comment this should be uppercase (and use ticks `) because they are referring to search terms within ACS which is uppercased.